validation/lightcurve/make.py

```diff
@@ -219,16 +225,25 @@ def make_summary(types):
     log.info("Making summary plots.")
     ax=None
     for type in types:
-        path = f"results/lightcurve_{type}.fits"
+        filename = make_path("results")
+        path = filename / f"lightcurve_{type}.fits"
+        filename.mkdir(exist_ok=True)
```

Comment:
Here it is not necessary to try to create the directory when reading the file. Remove.